src/components/ContinuationIn/ExtraproRegistration.vue

```diff
@@ -640,10 +640,8 @@ export default class ExtraproRegistration extends Mixins(DateMixin) {
 
   /** Emits form validity. */
   @Watch('isBusinessActive')
-  @Watch('business.homeJurisdiction')
-  @Watch('business.homeIncorporationDate')
+  @Watch('business', { deep: true })
```

Comment:
The old code assumed the form validity would change and catch any data changes, but that's not valid. For example, I could change a home identifier or home legal name character and the form validity wouldn't change, and also line 649 wouldn't be called, so the page wouldn't warn  you if you tried to leave with unsaved changes.

Instead of adding those extra watchers, in the new code I simply deep-watch the object (ie, all properties).

Reply:
Makes a lot of sense.